OpenPopUpContent.uc

```diff
@@ -0,0 +1,62 @@
+class OpenPopUpContent extends UWindowDialogClientWindow;
+
+var R6WindowWrappedTextArea wTextBox;
+var R6WindowButton butURL;
+var string urlString;
+var float fMargin;
+
+function MakeTextBox()
+{
+	wTextBox = R6WindowWrappedTextArea(CreateWindow(class'R6WindowWrappedTextArea',fMargin,fMargin,WinWidth-(fMargin*2),WinHeight-(fMargin*2),self));

```

Comment:
Do we need to double fMargin?

Reply:
The base CreateWindow() function in UWindowWindow.uc takes two arguments there, X and Y values.  Ubisoft, when building pop up boxes, used the same fMargin value (4.0) for both.  I kept it consistent with Ubisoft's implementation but of course we could have different values for the X margin and the Y margin.